MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+    }
+
+    public class Address
+    {
+        public int PostalCode { get; set; }
+        public string City { get; set; }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+#### AutoComplete editor modes
+
+The [DataFormAutoCompleteEditor]`Link` provides display the filtered suggestions.
+
+The `AutoCompleteMode` property is used to choose the suggestion display mode in the `DataFormAutoCompleteItem` class.
+
+#### AutoComplete editor suggestion options
+
+The phenomenon of string comparison for filtering suggestions can be changed using the "StartsWith" and "Containswith" `Link` property. The default filtering strategy is “StartsWith”, and it is case insensitive. The available filtering modes are,
+
+* StartsWith
+
+* ContainsWithCaseSensitive
+
+#### Filtering words that starts with the input text
+
+Displays all the matches that contain first character of the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+
+#### Filtering words that contains with the input text
+
+Displays all the matches that contain the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+	
+## Custom editor
+
+The custom editor can be added to DataForm by inheriting the `IDataFormEditor` class for business models. You can create custom editor using implement all method in IDataFormEditor. 
+
+To add custom editor in DataForm, register the editor with custom registered type for the required property using `RegisterEditor` method. You can also customize editor settings by using available methods in `IDataFormEditor`. 
+
+### Creating custom editor using IDataFormEditor 
+
+Views such as labels, buttons, and sliders can be loaded to custom editor. Here, entry is loaded as custom editor for `PhoneNumer` property. 
+
+### Below methioned example code for Numeric editor which accepts only numbers 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class NumericTextEditor : IDataFormEditor 
+    { 
+        public NumericEditor(Syncfusion.Maui.DataForm.SfDataForm dataForm)
+        {
+            this.dataForm = dataForm;
+        }
+
+        public View CreateEditorView(DataFormItem dataFormItem) 
+        { 
+            Entry inputView = new Entry();
+            if (dataFormItem.FieldName == "CVV")
+            {
+                inputView.IsPassword = true;
+                inputView.MaxLength = 3;
+            }
+            inputView.Keyboard = Keyboard.Numeric;
+            inputView.Placeholder = dataFormItem.PlaceholderText;
+            DataFormTextStyle textStyle = dataForm.EditorTextStyle;
+            inputView.TextColor = textStyle.TextColor;
+            inputView.FontSize = textStyle.FontSize;
+            inputView.FontFamily = textStyle.FontFamily;
+            inputView.FontAttributes = textStyle.FontAttributes;
+            inputView.TextChanged += this.OnViewTextChanged;
+            this.dataFormCustomItem = (DataFormCustomItem)dataFormItem;
```

Comment:
this.dataFormCustomItem => this variable not declared anywhere

Reply:
dataFormCustomItem added